tests/core/test_tracker_stores.py

```diff
@@ -368,3 +368,14 @@ def test_fail_safe_tracker_store_with_retrieve_error():
 
     assert tracker_store.retrieve("sender_id") is None
     on_error_callback.assert_called_once()
+
+
+def test_set_fail_safe_tracker_store_domain(default_domain: Domain):
+    tracker_store = InMemoryTrackerStore(domain)
+    fallback_tracker_store = InMemoryTrackerStore(None)
+    failsafe_store = FailSafeTrackerStore(tracker_store, None, fallback_tracker_store)
+
+    failsafe_store.domain = default_domain
+    assert failsafe_store.domain == default_domain
```

Comment:
let's check for identity
```suggestion
    assert failsafe_store.domain is default_domain
```